turbine/src/broadcast_stage.rs

```diff
@@ -56,7 +56,9 @@ mod standard_broadcast_run;
 const CLUSTER_NODES_CACHE_NUM_EPOCH_CAP: usize = 8;
 const CLUSTER_NODES_CACHE_TTL: Duration = Duration::from_secs(5);
 
-pub(crate) const NUM_INSERT_THREADS: usize = 2;
+// Blockstore::insert_threads() obtains and holds a write lock for the entire function. Until
+// this changes, there is no point in having more than one blockstore inserter thread.
+pub(crate) const NUM_INSERT_THREADS: usize = 1;
```

Comment:
Should we remove this constant entirely and simplify:
https://github.com/anza-xyz/agave/blob/cde467dc8/turbine/src/broadcast_stage.rs#L334-L352

Reply:
I could go either way. Hypothetically, if we remove the lock in Blockstore, more threads could potentially be beneficial. But, given that Blockstore isn't the bottleneck, I'm not sure we'll spend much time on it for a while so the simplification you suggested could be good